main_logic/tts_client.py

```
@@ -990,3 +1007,241 @@ def get_tts_worker(core_api_type='qwen', has_custom_voice=False):
+        current_speech_id = None
+        # ready_sent = False #初始化就绪信号标志
+        # CosyVoice3 默认采样率通常为 24000Hz (如果是 CosyVoice1 则为 22050Hz)
+        # 你的 server 代码加载的是 Fun-CosyVoice3-0.5B，所以这里设定为 24000
+        SRC_RATE = 24000
+        # 重采样？ 调用soxr.ResampleStream 参数分别为 原采样率 ,重采样后的采样率 num_channels是声道数？
+        resampler = soxr.ResampleStream(SRC_RATE, 48000, 1, dtype='float32')
+
+        # === 内部辅助函数：连接/重连 ===
+        async def ensure_connection():
+            nonlocal ws, receive_task
+            # 如果已有连接，先尝试关闭
```

Ruff (0.14.10)

Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)